IABot/Parser/zhwiki.php

```diff
@@ -140,8 +137,9 @@ protected function noRescueLink( &$link, &$modifiedLinks, $tid, $id ) {
         } else {
             $link['newdata']['tag_type'] = "template";
             $link['newdata']['tag_template']['name'] = "dead link";
-            $link['newdata']['tag_template']['parameters']['date'] = self::strftime( '%B %Y' );
+            $link['newdata']['tag_template']['parameters']['date'] = self::strftime( '%Y年%-m月' );
             $link['newdata']['tag_template']['parameters']['bot'] = USERNAME;
+            $link['newdata']['tag_template']['parameters']['fix-attempted'] = 'yes';
```

Comment:
Why is this being added?  Your template doesn't support that.

Reply:
The support [has added](https://zh.wikipedia.org/w/index.php?title=Template:Dead_link&diff=46909137&oldid=44524550) on zhwiki.